Sources/PodToBUILD/GlobNode.swift

```diff
@@ -155,6 +155,47 @@ extension Either: SkylarkConvertible where T == Set<String>, U == GlobNode {
             return globVal.toSkylark()
         }
     }
+
+    public func simplify() -> Either<Set<String>, GlobNode> {
+        // Recursivly simplfies the globs
+        switch self {
+        case let .left(val):
+            // Base case, this is as simple as it gets
+            return .left(val)
+        case let .right(val):
+            let include = val.include.simplify()
+            let exclude = val.exclude.simplify()
+            if exclude.isEmpty {
+                // When there is no excludes we can do the following:
+                // 1. smash all sets into a single set
+                // 2. return a set if there are no other globs
+                // 3. otherwise, return a simplified glob with 1 set and
+                // remaining globs
+                var setAccum: Set<String> = Set()
+                let remainingGlobs = include
+                    .reduce(into: [Either<Set<String>, GlobNode>]()) {
+                    accum, next in
+                    switch next {
+                    case let .left(val):
+                        setAccum = setAccum <> val
+                    case let .right(val):
+                        if !val.isEmpty {
+                            accum.append(next)
+                        }
+                    }
+                }
+
+                // If there are no remaining globs, simplify to a set
+                if remainingGlobs.count == 0 {
```

Comment:
`remainingGlobs.isEmpty` ?

Reply:
This line is recusivly magic - I believe the input to the `reduce` function should have already been filtered via `[].simplify()` so no need to check 2x. An array of simplified `Either<Set,Glob>`'s should have a count of `0`